examples/meanvar.jl

```diff
@@ -0,0 +1,44 @@
+# Welford's algorithm
+#
+# Welford, B. P. (1962). "Note on a method for calculating corrected sums of squares and products". Technometrics. 4 (3): 419–420. doi:10.2307/1266577. JSTOR 1266577
+
+
+using Transducers
+const T = Transducers
```

Comment:
Let's use

```suggestion
using Transducers: R_, start, next, complete, inner, xform, wrap, unwrap, wrapping
```

for consistency with other examples.